recipe/meta.yaml

```
@@ -25,28 +28,29 @@ requirements:
   host:
     # Host dependencies are pinned to last build
     - python
-    - cython 3.0.8
+    - cython >=3.0.10
     - numpy {{ numpy }}
     - llvm-openmp 14.0.6  # [osx]
     - pip
-    - scipy >=1.6.0
-    - setuptools
-    - wheel
+    - scipy =1.9.3        # [py<312]
+    - scipy =1.11.3       # [py>=312]
+    - meson-python >=0.15.0
+    - python-build
     # mkl variant pulls in intel-openmp which conflicts with llvm-openmp. i.e. force to use openblas variant of numpy, scipy, etc.
-    - blas=*=openblas  # [osx and x86_64]
+    - blas * openblas     # [osx and x86_64]
   run:
     - python
     - joblib >=1.2.0
     - {{ pin_compatible('numpy') }}
     - scipy >=1.6.0
```

this needs a selector
```suggestion
    - scipy >=1.6.0  # [not (osx and x86_64)]
```